omero/developers/Modules/Search.rst

```diff
@@ -57,10 +66,20 @@ FullTextAnalyzer <src/main/java/ome/services/fulltext/FullTextAnalyzer.java>`.
 
 Assuming these entries above for Image.name:
 
--  searching for **GFP-H2B** returns 1 and 2.
--  searching for **"GFP H2B"** also returns 1 and 2.
--  searching for **GFP H2B** returns 1, 2, and 3, since the two terms
+-  searching for **GFP-H2B** returns 1, 2, 3 and 4, because of the tokenizing on the hyphen and joining of the tokens by an **OR**. This behavior is changed by adding wildcards or quotations, see below.
```

Comment:
I'm not sure you need "This behaviour is changed... " sentence since that's true for all queries.